easybuild/easyblocks/s/slepc.py

```diff
@@ -78,7 +78,7 @@ def configure_step(self):
         self.log.debug('SLEPC_DIR: %s' % os.getenv('SLEPC_DIR'))
 
         # optional dependencies
-        depfilter = self.cfg.builddependencies() + ["PETSc"]
+        depfilter = self.cfg.builddependencies() + ['PETSc', 'Python']
         deps = [dep['name'] for dep in self.cfg.dependencies() if not dep['name'] in depfilter]
```

Comment:
@lexming Please change this to `if dep['name'] not in dep_filter` while you're at it

Reply:
fixed :+1: